Controllers/TaskController.php

```diff
@@ -0,0 +1,114 @@
+<?php
+
+namespace Controllers;
+
+use Models\TaskModel;
+
+class TaskController implements IController
+{
+    private $model;
```

Comment:
Please name it more understandable. What **model**? BMW? Fiat? What the purpose if this model?